predicators/approaches/maple_q_approach.py

```diff
@@ -108,11 +112,6 @@ def _learn_nsrts(self, trajectories: List[LowLevelTrajectory],
                      annotations: Optional[List[Any]]) -> None:
         # Start by learning NSRTs in the usual way.
         super()._learn_nsrts(trajectories, online_learning_cycle, annotations)
-        # Check the assumption that operators and options are 1:1.
-        # This is just an implementation convenience.
-        assert len({nsrt.option for nsrt in self._nsrts}) == len(self._nsrts)
-        for nsrt in self._nsrts:
-            assert nsrt.option_vars == nsrt.parameters
```

Comment:
we definitely no longer want this assumption, but we also don't want silent failures when the old code for our Planning to PRactice paper is run. Could you put back in this code, but put it under an `if` statement that says `if CFG.approach == "active_sampler_learning"`?